abs.yaml

```diff
@@ -0,0 +1 @@
+aggregate_check: false
```

Comment:
I'm curious why we're going this route? More importantly, why do we need to push these changes into a branch other than `main`?

Reply:
Because our current version is 2.3.0.

It seems that the rebuilding aims to build **py312** artifacts for linux-64:
https://repo.anaconda.com/pkgs/main/linux-64
```
pytorch-lightning-2.0.3-py310h06a4308_0.tar.bz2
pytorch-lightning-2.0.3-py311h06a4308_0.tar.bz2
pytorch-lightning-2.0.3-py38h06a4308_0.tar.bz2
pytorch-lightning-2.0.3-py39h06a4308_0.tar.bz2
```

Reply:
I added `aggregate_check: false` to avoid having an extra submodule in `aggregate` specifically for this older version, since we will likely only rebuild it once.

Reply:
My deepest apologies! My dyslexia kicked in there for a bit, because I thought it was the same version with just a bumped build number. Thanks!

Reply:
No worries! I had the same confusion initially because of the similar version numbers.